README.md

```diff
@@ -1,10 +1,113 @@
 # mrz-parser
+
+[![Build Status](https://travis-ci.com/Coppernic/mrz-parser.svg?branch=master)](https://travis-ci.com/Coppernic/mrz-parser)
+
 MRZ parser in Kotlin
 
-- TD1
-- TD2
-- TD3
-- ID Card
-- Passport
-- Visa
-- 
\ No newline at end of file
+## Motivation
+
+This library aims to provide an API in kotlin to parse MRZ of travel and identity documents.
+
+## Download
+
+Include instructions on how to integrate the library into your projects. For instance install in your build.gradle:
```

Comment:
Do you mean here that you will later include instructions? Can you say in french what do you mean?